providers/azure/resources/azure.lr.manifest.yaml

```diff
@@ -953,16 +953,35 @@ resources:
   azure.subscription.networkService:
     fields:
       bastionHosts: {}
+      firewallPolicies: {}
+      firewalls: {}
       interfaces: {}
+      loadBalancers: {}
+      natGateways: {}
       publicIpAddresses: {}
       securityGroups: {}
+      subnets: {}
       subscriptionId: {}
+      virtualNetworkGateways: {}
+      virtualNetworks: {}
       watchers: {}
     is_private: true
     min_mondoo_version: latest
     platform:
       name:
       - azure
+  azure.subscription.networkService.backendAddressPool:
+    fields:
+      etag: {}
+      id: {}
+      name: {}
+      properties: {}
+      type: {}
+    is_private: true
+    min_mondoo_version: latest
```

Comment:
Can we get the intended version on all these?

Reply:
I updated it, but we should revisit this whole approach. This should switch to provider version and not cnquery version